### DISCUSSION TITLE
feature request: separate cut and paste for ranges within blocks

### DISCUSSION THREAD

**Discussion by benkoshy** [retypeapp/retype#413] [Ideas]
Retype allows you do things like highlight specific lines.  The cut and paste feature on code blocks are nice. However, we need to be able to cut and paste specific regions/lines. 

I prefer not to require my users to manually cut and paste / highlight specific lines. Not critical or urgent, just a 'nice to have' feature.

Notice how there is only one <kbd> Ctrl </kbd> +  <kbd> V </kbd> option for the entire block? We need separate ones:

![image](https://user-images.githubusercontent.com/15097447/187857262-a4715fdd-5fff-4b76-a759-919359557300.png)

Separating them, to allow users to cut and paste only a few specific lines is a possibility, but is not as aesthetically pleasing:

![image](https://user-images.githubusercontent.com/15097447/187858277-d2426136-b821-49da-a32b-320315bfd7e5.png)

Source: https://benkoshy.github.io/pagy/docs/how-to/#customize-the-link-attributes

**Comment by benkoshy** [comment]
Please correct me if it's already there etc. I didn't see it in the documentation: https://retype.com/components/code-block/

**Comment by DannyBen** [comment]
What's wrong with multiple blocks?

**Reply by benkoshy** [reply]
Not aesthetically pleasing. See above for comparison. 

maybe it's worth the effort for the maintainer, maybe not.

I figure all feedback is valuable. even bad ideas. all i can say is that if the feature existed, I would happily make use of it........the effort required to type out a feature request is proof of my desire.

the devil is in the details and user feedback is what drives the quality. the more feedback, the better position the maintainers will be in to improve their offering. the nuances are what make products super nice.

my two cents.
